kingfisher_scrapy/spidermiddlewares.py

```diff
@@ -269,13 +270,14 @@ async def process_spider_output(self, response, result, spider):
 
 class RetryDataErrorMiddleware:
     """
-    Retries a request for a ZIP file up to 3 times, on the assumption that, if the spider raises a ``BadZipFile``
-    exception, then the response was truncated.
+    Retries a request up to 3 times. Either when the spider raises a ``BadZipFile`` exception, on the assumption that
+    the response was truncated, or when the spider raises a ``RetryableBadResponseError`` exception on the assumption
+    that the error was temporary.
```

Comment:
No need to assume in second case, as the exception is raised deliberately.

```suggestion
    Retries a request up to 3 times. Either when the spider raises a ``BadZipFile`` exception, on the assumption that
    the response was truncated, or when the spider raises a ``RetryableBadResponseError`` exception.
```